pom.xml

```diff
@@ -19,11 +19,6 @@
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <parent>
-        <groupId>org.jboss</groupId>
-        <artifactId>jboss-parent</artifactId>
-        <version>39</version>
-    </parent>
```

Comment:
Is there a reason we don't just use the productized version of the parent?

hmm... ...or maybe there isn't one https://github.com/jbossas/jboss-eap8/blob/1e4739ce823222c263c085ffbec2ffbb76c5c1a4/pom.xml#L26-L32.

That said, if we don't want to use a parent I think there are going to be several plugins we need to override. For example the `maven-compiler-plugin` so we compile to the correct bytecode version.